app/scripts/controllers/metametrics.js

```diff
@@ -754,7 +754,7 @@ export default class MetaMetricsController {
   }
 
   /**
-   * Returns an array of all of the collectibles/NFTs the user
+   * Returns an array of all of the nfts/NFTs the user
```

Comment:
This appears to have been offering two names for the same thing, but now that there is just one name I think you can say:

``` suggestion
   * Returns an array of all of the NFTs the user
```